editor/editor_themes.cpp

```diff
@@ -367,6 +367,10 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	theme->set_color("box_selection_fill_color", "Editor", accent_color * Color(1, 1, 1, 0.3));
 	theme->set_color("box_selection_stroke_color", "Editor", accent_color * Color(1, 1, 1, 0.8));
 
+	theme->set_color("axis_x_color", "Editor", Color(0.96, 0.20, 0.32));
```

Comment:
We might be duplicating our effort a bit, I abstracted some color logic out of editor_properties into editor_inspector https://github.com/godotengine/godot/pull/32068/files#diff-554c9c97362d5a6f63b7244253542ffeR698

Might be worth placing all of this logic together. I think the axis colors are the same as the inspector colors, if not we could fix that inconsistency too.